package.json

```diff
@@ -69,7 +70,7 @@
     "dist-src": "cp -r ./src ./lib && find ./lib -name '__tests__' -type d -exec rm -r '{}' + && ./node_modules/.bin/babel lib --out-dir lib",
     "livereload": "NODE_ENV='development' ./node_modules/.bin/gulp docs:livereload",
     "npm-publish": "npm publish ./",
-    "test": "jest",
+    "test": "mv .babelrc .babelrc-tmp && jest && mv .babelrc-tmp .babelrc",
```

Comment:
If one test fails, this will continue to fail until the `.babelrc` is moved back